pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -331,6 +331,19 @@ public class ServiceConfiguration implements PulsarConfiguration {
             + "(0 to disable limiting)")
     private int maxHttpServerConnections = 2048;
 
+    @FieldContext(category = CATEGORY_SERVER, doc =
```

Comment:
Can we make this configuration dynamic so that we can update the exclude paths without restart the brokers?

Reply:
@hangc0276 I'd suggest a separate change for that since making this dynamic will be a significant change to the current PR.